server/project/WebAssetsBundler.scala

```diff
@@ -76,10 +71,10 @@ object WebAssetsBundler extends AutoPlugin {
     }
   )
 
-  /** Given list of all assets (css, image, TS files) figures out whether any TS
-    * files have been changed and recompiles them. If no files have been changed
-    * assumes JS and CSS files have been compiled at earlier iterations and
-    * returns them.
+  /** Given list of all assets (css, image, TS, Sass files) figures out whether
+    * any TS files have been changed and recompiles them. If no files have been
```

Comment:
suggestion (blocking):  It should probably say "whether any TS **or Sass** files have been changed...".